lib/utils.py

```diff
@@ -202,7 +202,11 @@ def decompress_files(name, dest_folder):
     else:
         raise RuntimeError('Unknown file format "{}"'.format(name))
     tar = fobject(name, mode=modestr)
-    tar.extractall(dest_folder)
+    try:
+        tar.extractall(dest_folder)
+    except IOError:
```

Comment:
Can you do something like [this](https://github.com/ruiztulio/backupws/blob/master/lib/utils.py#L582) and check if the error is really because of no free space?